src/types/Comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface Comment {
+    postId: number,
+    id: number,
+    name: string,
+    email: string,
+    body: string,
+  }
```

Comment:
Add a new line 
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file